src/main/java/org/ibp/api/java/impl/middleware/study/StudyGermplasmSourceServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package org.ibp.api.java.impl.middleware.study;
+
+import org.generationcp.middleware.service.api.PedigreeService;
+import org.generationcp.middleware.service.api.study.germplasm.source.StudyGermplasmSourceDto;
+import org.generationcp.middleware.service.api.study.germplasm.source.StudyGermplasmSourceRequest;
+import org.generationcp.middleware.util.CrossExpansionProperties;
+import org.ibp.api.java.study.StudyGermplasmSourceService;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.Resource;
+import java.util.List;
+
+@Service
+@Transactional
+public class StudyGermplasmSourceServiceImpl implements StudyGermplasmSourceService {
```

Comment:
Rename to GermplasmStudySourceServiceXXX

Reply:
Done